gslib/tests/test_naming.py

```diff
@@ -1429,7 +1430,7 @@ def testRemoveObjsCommand(self):
       self.RunCommand('rm', [suri(dst_bucket_uri, 'non_existent')])
       self.fail('Did not get expected CommandException')
     except CommandException as e:
-      self.assertIn(NO_URLS_MATCHED_GENERIC, e.reason)
+      self.assertIn(NO_URLS_MATCHED_PREFIX, e.reason)
```

Comment:
Why use NO_URLS_MATCHED_PREFIX here? I am guessing just NO_URLS_MATCHED_PREFIX msg will not be raised from anywhere?

Reply:
Made the error we're testing for more specific.